package.js

```diff
@@ -80,7 +80,6 @@ Package.onUse(function (api) {
     'sass/components/_variables.scss',
     'sass/components/_waves.scss',
     'sass/components/_carousel.scss',
-    'sass/materialize.scss',
```

Comment:
fourseven:scss compiles this file automatically. Removing it will default to no css compilation at all. The user must add it manually to any scss file with no preceding '_'